index.js

```diff
@@ -113,6 +113,29 @@ function mockService(service) {
   services[service].stub = serviceStub;
 };
 
+/**
+ * Wraps a sinon stub or jest mock function as a fully functional replacement function
+ */
+function wrapTestStubReplaceFn(replace) {
+  if (typeof replace !== 'function' || !(replace._isMockFunction || replace.isSinonProxy)) {
+    return replace;
+  }
+
+  return (params, cb) => {
+    let result;
+    try {
+      result = replace(params);
```

Comment:
It is better to use 'const' instead of 'let'.

```js
const result = replace(params);
```

Reply:
Updated on branch. Thanks @abetomo